package.json

```diff
@@ -11,6 +11,7 @@
     "node": ">=5"
   },
   "scripts": {
+    "postinstall": "cd lighthouse-core && npm install && npm run build",
```

Comment:
Curious what happens with the typescript when someone runs `npm install lighthouse`? Do you precompile it and then that gets pushed to npm?